### PR TITLE
Template spec solution package update for Trend Micro Apex One

### PR DESCRIPTION
**Pull request by vmanojreddy**
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Trend Micro Apex One.
See guidance below
Version Updated:
2.0.0

See guidance below
Testing Completed:
The link for test results is -
https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7BE92B606B-3CF6-4371-B2CC-E383BE74366E%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20Trend%20Micro%20Apex%20One.docx&action=default&mobileredirect=true&share=IQFrYCvp9jxxQ7LM44O-dDZuAUSbkzR69nQBISkEHeIu-uc
See guidance below
This is a 3P solution.

### CODE REVIEW

File: Solutions/Trend Micro Apex One/Data Connectors/TrendMicro_ApexOne.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "id": "TrendMicroApexOne",
```
Comment (v-amolpatil) [inline Solutions/Trend Micro Apex One/Data Connectors/TrendMicro_ApexOne.json:2]: Please rename all occurrences in this file `Data Connectors/TrendMicro_ApexOne.json` of `Azure Sentinel` to `Microsoft Sentinel`

**Comment by v-amolpatil** [comment]
Please update `isPreview` key value to `false` under `Data Connectors/TrendMicro_ApexOne.json` file.
Currently it looks like below:

`    "availability": {
        "status": 1,
        "isPreview": true
    },`

**Comment by vmanojreddy** [comment]
> Please update `isPreview` key value to `false` under `Data Connectors/TrendMicro_ApexOne.json` file. Currently it looks like below:
> 
> ` "availability": { "status": 1, "isPreview": true },`

@v-amolpatil Done

**Review by v-amolpatil — APPROVED** [review]
All changes looks good to me.

**Pull request merged**